docs/models/index.rst

```diff
@@ -427,6 +427,42 @@ the model to be fit would be given by an expression like::
 
    >>> mdl = g1 + g2 + 0 * sep
 
+Complex functional relationships
```

Comment:
Why are we getting this change (and the other changes in this file) in this PR?

Reply:
No. That's from the already merged #1653. Not sure why GH shows this as new code here. But I will probably reabse / squash commits a couple more times, so I hope that problem goes away by itself.